### ISSUE TITLE
'Folder Sets' do not create when using "/" in 'Set Name'

### ISSUE THREAD

**Issue by vizguy** [six7/figma-tokens#945]
**Describe the bug**
When creating set names, we are using (a/b) - just like we would in Figma to create 'style folders'.

These accordion folders do not show up when using this naming structure.  We are wondering if the 111 Gitlab fix did something to the folder organization within the plugin.

**To Reproduce**
1. Create a set.
2. Name the set (a/b)
3. Create another set.
4. Name the new set (a/c)
5. Theoretically that should create a folder in the Figma Tokens plugin and it does not.

**Expected behavior**
Folder is created in the FT UI

**Comment by abinashmohanty** [comment]
This is so true. I use _jsonbin_ though and I realised, whenever I try to create a sub-category using / or . this set completely disappears from my token set, but if I try to recreate the sub-category with the same name, it says _name already exists_. 

For example, I've added a primary color token and a sub-category like these.

**primary** > main category > it stays
**primary.alpha.100** > sub category > it disappears 
Create another set > **primary.alpha.100** > sub category > it says _name already exists_ .

This also happens when we try to edit a few token, token group, etc.. from category to sub-category and it doesn't show within the plugin widget but somehow stays within its code library resulting the status _already exists_. 

Should have option to purge / clean up option.
